app.py

```diff
@@ -44,7 +44,7 @@ def static_file(path):
 SHOULD_STREAM = True if AZURE_OPENAI_STREAM.lower() == "true" else False
 
 def is_chat_model():
-    if 'gpt-4' in AZURE_OPENAI_MODEL_NAME.lower():
+    if 'gpt-4' in AZURE_OPENAI_MODEL_NAME.lower() or 'gpt-35' in AZURE_OPENAI_MODEL_NAME.lower():
```

Comment:
Let's revert this change - we want to keep using completions for Turbo for better AIQ in the with-data case for now. There was a bug with Turbo models named "chat" which has been fixed.